search/releases/7.2/index.adoc

```diff
@@ -0,0 +1,163 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: search
+:awestruct-series_version: "7.2"
+:page-interpolate: true
+:latest-release-version: #{series(page, page.series_version).releases.first.version}
+:hsearch-doc-url-prefix: #{reference_doc(site.projects[page.project], series(page, page.series_version)).html_url}
+
+=== Dependency upgrades
+
+[[orm-version]]
+Hibernate ORM (link:{hsearch-jira-url-prefix}/HSEARCH-5150[HSEARCH-5150])::
```

Comment:
I would recommend dropping the references to Jira issues here. They will likely be painful to maintain as you release new alphas/betas, and don't bring much in this particular case -- as opposed to the blog post which is specifically about the new alpha.